pkg/virt-controller/services/template.go

```diff
@@ -1049,6 +1049,10 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 		if nonRoot {
 			command = append(command, "--run-as-nonroot")
 		}
+		if customDebugFilters, exists := vmi.Annotations["log-filters"]; exists {
```

Comment:
Please define this annotation with already used convention [here](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/types.go#L637)

Reply:
Thanks! will do!